src/main/java/TaskList.java

```diff
@@ -0,0 +1,167 @@
+public class TaskList {
+    private Task[] taskArray = new Task[100];
+    private int totalTaskNum = 0;
+
+    public boolean addTask(String userInput){
+        String[] userInputSplited = userInput.split("/");
+        String[] userCommand = userInputSplited[0].split(" ");
+        switch(userCommand[0]){
+            case "todo":
+                return addTodo(userInputSplited);
+            case "deadline":
+                return addDeadline(userInputSplited);
+            case "event":
+                return addEvent(userInputSplited);
+            default:
+                System.out.println("Failed to add: Invalid Task format"); return false;
```

Comment:
This might be better according to switch statement coding standards
```
default:
    System.out.println("Failed to add: Invalid Task format"); 
    return false;
```